tests/ambsym/tresolution_identifier_2_ok.nim

```diff
@@ -0,0 +1,17 @@
+discard """
+labels: "enum identifier proc resolution"
+description: '''
+  . mresolution_identifier1 defines TExport, an enum with x, y, z.
+  . mresolution_identifier2 defines TExport, an enum with a, b, c.
+  . By expliciting a namespace alongside the type name, we are able to
+    resolve the ambiguity.
+  . This goes for both the TExport and the `foo` call. The `foo` call should
+    be resolved to the one defined in identifier_2, *not* in identifier_2_copy.
```

Comment:
adding a `foo` proc to mresolution_identifier_1 seems like a good idea. 
Anyway, this whole category reads like a spec, and is probably made redundant by those. Is this the case, @haxscramper ?